internal/provider/data_source_address_lot_test.go

```diff
@@ -0,0 +1,61 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+type dataSourceNetworkingAddressLotConfig struct {
+	BlockName string
+}
+
+var datasourceNetworkingAddressLotsConfigTpl = `
+data "oxide_networking_address_lots" "{{.BlockName}}" {
+	timeouts = {
+	  read = "1m"
+	}
+}
+`
+
+func TestAccCloudDataSourceNetworkingAddressLots_full(t *testing.T) {
```

Comment:
The prefix is to differentiate between "cloud" and "silo" endpoints. I use this manly for testing on the colo rack, where the user on the GH action doesn't have admin privileges

```suggestion
func TestAccSiloDataSourceNetworkingAddressLots_full(t *testing.T) {
```